src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -337,6 +339,12 @@
       url: /deprecated-components/filter-list
     - title: Filtered search
       url: /deprecated-components/filtered-search
+    - title: Inline autocomplete
+      url: /deprecated-components/inline-autocomplete
+    - title: Markdown editor
+      url: /deprecated-components/markdown-editor
+    - title: Markdown viewer
+      url: /deprecated-components/markdown-viewer
```

Comment:
These have been moved to Primer Recipes and don't need to be maintained in the Primer core docs.

```suggestion
```